app/models/host.rb

```diff
@@ -7,7 +7,7 @@ class Host < ApplicationRecord
   OS_MAJOR_VERSION = Arel.sql("system_profile->'operating_system'->'major'")
 
   SORTABLE_BY = {
-    name: :display_name,
+    name: Arel.sql('LOWER(display_name)'),
```

Comment:
```
Host.arel_table[:display_name].lower
```

Reply:
That is not compatible with our hash-sorting format that is being built in the controller :thinking: